WordPress/Classes/ViewRelated/Post/PostListViewController.swift

```diff
@@ -25,6 +25,10 @@ class PostListViewController : AbstractPostListViewController, UIViewControllerR
     @IBOutlet private var imageCellForLayout: PostCardTableViewCell!
     @IBOutlet private weak var authorFilterSegmentedControl: UISegmentedControl!
 
+    @IBOutlet var authorsFilterView : UIView! // Search lives here on iPad
```

Comment:
With this comment, it might be helpful with a more context, such as why it lives here on iPad or where it lives on iPhone instead.